### PR TITLE
Mcs 844 cache addressables docs

### PR DESCRIPTION
**Pull request by kdrumm**
Adding some documentation and instructions to cache addressables script.

### CODE REVIEW

**Review by JuanRMarquez-Ortiz — APPROVED** [review]

File: scripts/cache_addressables.py [hunk 1]
```diff
@@ -1,6 +1,22 @@
 import os
 import sys
 
+'''
+Script will start Unity build with a flag to clear the addressables cache,
+cache all the data in catalog and then close the application.
+
+Usage: python cache_addressables.py <unity-build-path>
```
Comment (deanwetherby) [inline scripts/cache_addressables.py:8]: If a researcher/developer installs MCS from PyPI, do they have access to this script?

**Review by deanwetherby — COMMENTED** [review]
I think the cache_addressables script needs to be added to console_scripts in setup.py that way it's available in venv/bin/ to run from the command line.

File: scripts/cache_addressables.py (hunk 1 above)
Comment (deanwetherby) [reply scripts/cache_addressables.py:8]: Confirmed that the only scripts in the venv when installing from PyPI are those specified in the console_scripts. Those scripts are installed in venv/bin/.

File: setup.py [hunk 2]
```diff
@@ -47,7 +47,8 @@ def get_version():
     entry_points={
         'console_scripts': [
             'run_in_human_input_mode=scripts.run_human_input:main',
-            'run_scene_timer=scripts.run_scene_timer:main'
+            'run_scene_timer=scripts.run_scene_timer:main',
+            'cache_addressables=scripts/cache_addressables:main'
```
Comment (deanwetherby) [inline setup.py:51]: Adding the script to entry_points will put the script in venv/bin when installed from PyPI. With an activate python virtual environment, the developer could call the script from the command line via:

`cache_addressables <MCS_path>`

Without explicitly adding this script to entry_points, an MCS installation from PyPI will not install the script.

Comment (rartiss55) [reply setup.py:51]: Good catch!

Comment (rartiss55) [reply setup.py:51]: If this is the case, the line about this script in install.rst should be updated too.

**Review by JuanRMarquez-Ortiz — APPROVED** [review]

**Review by rartiss55 — APPROVED** [review]

**Pull request merged**